docs/manual/working/javaGuide/main/templates/JavaTemplates.md

```diff
@@ -110,6 +110,15 @@ Or even several parameter groups:
 ```scala
 @(title:String)(body: Html)
 ```
+### Defining an implicit Requests in Templates
+
+You can define implicit parameters for a template too:
+
+```
+@(user: models.User)(implicit requestHeader: RequestHeader, messagesProvider: MessagesProvider)
```

Comment:
I don't think this example belongs here. This is the Java documentation and you're referencing Scala API classes from Play.

Reply:
I'll remove this. But I am not sure which example is correct for Java documentation.